x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/sample_logs_input.tsx

```diff
@@ -205,9 +79,6 @@ export const SampleLogsInput = React.memo<SampleLogsInputProps>(({ integrationSe
               <EuiText size="s" textAlign="center">
                 {i18n.LOGS_SAMPLE_DESCRIPTION}
               </EuiText>
-              <EuiText size="xs" color="subdued" textAlign="center">
-                {i18n.LOGS_SAMPLE_DESCRIPTION_2}
```

Comment:
I think we can keep the message for now, since we didn't add the code that would _actually_ support it, e.g. parse syslog.

Reply:
Can check with UI experts here. @semd Do you recommend to have this in UI , although we show an error message for those log samples that are not implemented yet.

Reply:
Yes, I agree with @ilyannn. We should always inform the user in advance about the supported formats.

Reply:
Thanks. Left this until we have other formats supported